check_color_from_pair_number.py

```diff
@@ -0,0 +1,19 @@
+
+import main
+
+def get_color_from_pair_number(pair_number):
```

Comment:
production code and test code is part of the same file. It is better to keep test code separate from production code. There is a possibility of accidental execution of test code after getting shipped to field as part of production code.